packages/1xdev-users/__test__/get-users.spec.tsx

```diff
@@ -3,6 +3,6 @@ import { getUsers } from '../src';
 describe('getUsers', () => {
   it('Should get users', () => {
     const users = getUsers();
-    expect(users).toHaveLength(2);
+    expect(users).toHaveLength(3);
```

Comment:
Let's verify that your name is part of the items in the array, not only that the array has 3 items